src/jira.py

```diff
@@ -0,0 +1,144 @@
+import base64
+import re
+import requests
+import pprint
```

Comment:
Don't think you use pprint or time. If not, please remove.

Reply:
removed